Script/AtomicEditor/editor/Editor.ts

```diff
@@ -170,6 +170,37 @@ class Editor extends Atomic.ScriptObject {
     }
 
     /**
+     * Return a preference value or the provided default from the global user settings file
+     * @param  {string} groupName name of the section the preference lives under
+     * @param  {string} preferenceName name of the preference to retrieve
+     * @param  {number | boolean | string} defaultValue value to return if pref doesn't exist
+     * @return {number|boolean|string}
+     */
```

Comment:
What is your secret to these doc strings?  It is everything I can muster sometimes to even do a /// one for C++ methods!